packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -59,10 +64,10 @@ module.exports = {
     //
     // Useful for deploying to a public network.
     // Note: It's important to wrap the provider as a function to ensure truffle uses a new provider every time.
-    // ropsten: {
-    //   provider: () => new HDWalletProvider(mnemonic, `https://ropsten.infura.io/v3/YOUR-PROJECT-ID`),
-    //   network_id: 3,       // Ropsten's id
-    //   gas: 5500000,        // Ropsten has a lower block limit than mainnet
+    // goerli: {
+    //   provider: () => new HDWalletProvider(mnemonic, `https://goerli.infura.io/v3/${projectId}`),
+    //   network_id: 5,       // Goerli's id
+    //   gas: 5500000,        // Goerli has a lower block limit than mainnet
```

Comment:
Does the comment about gas hold true for Goerli?

Reply:
I'm not sure if it does and maybe this is not very helpful in here. What do you think? Should we just get rid of this `gas` line altogether?

Reply:
I couldn't find any suitable documentation for this `gas` line. I guess we can just get rid of this for now.

Reply:
@gnidan @haltman-at Do you guys have any insight on where this `gas` comment is from? Should it be kept / changed now that it's goerli?